bugwarrior/services/logseq.py

```diff
@@ -181,16 +181,16 @@ def get_formatted_title(self):
 
     # get a list of tags from the task content
     def get_tags_from_content(self):
-        # this includes #tagname, but ignores tags that are in the #[[tag name]] format
+        # pattern match for #[[multi word]] tags and #single word tags
+        # but ignore any non-tag use of the # character in URLs
+        # like http://example.com/page#test or in `#code`
+        # Regex Pattern: (?<=\s)#【.*】|(?<=\s)#\S+
+        # Note that this is processed after the content is unescaped,
+        # so we use the char_open_link and char_close_link
         tags = re.findall(
-            r"(#[^" + self.config.char_open_link + r"^\s]+)",
+            r"(?<=\s)#" + self.config.char_open_link + r".*?" + self.config.char_close_link + r"|(?<=\s)#\S+",
```

Comment:
I believe this should make CI happy:

```suggestion
            r"(?<=\s)#" + self.config.char_open_link + r".*?" + self.config.char_close_link
            + r"|(?<=\s)#\S+",
```

Reply:
With the updated regex from below I'd suggest the following line break to keep with in the line length and maintain readability
```python
        tags = re.findall(
            r"(?<=\s)"
            + r"(#" + self.config.char_open_link + r".*?" + self.config.char_close_link
            + r"|#\S+)",
            self.get_formatted_title()
        )
```